frontend/src/components/notification/notification-dropdown.tsx

```diff
@@ -46,16 +47,18 @@ const NotificationDropdown = ({user, token}: UserProps) => {
 
         fetchUserNotification();
 
-    }, []);
+    }, [token]);
     return (
-        <div>
+        <div className="flex flex-row gap-4">
             {user.role == "admin" || user.role == "moderator" || user.role == "analyst" ? (
                 <DropdownMenu>
                     <DropdownMenuTrigger asChild>
-                        <Button variant="ghost">
-                            <BellIcon className="h-4 w-4 mr-2" />
-                            Article Queue
+                        <Button variant="ghost" className="relative">
+                            <ShieldAlert
+                             className="h-4 w-4 mr-2" />
+                            <p className="hidden xl:block">Article Queue</p>
                             {queueNotifications.length ? ` (${queueNotifications.length})` : ""}
+                            <span className="absolute top-1 right-1 transform translate-x-1/2 -translate-y-1/2 h-4 w-4 rounded-full bg-red-500" />
```

Comment:
Display conditionally, reduce sizing and move close to text

```suggestion
                            {queueNotifications.length ? (
                                <span className="absolute top-1 right-1 transform translate-x-1/12 -translate-y-1/12 h-2 w-2 rounded-full bg-red-500" />
                            ) : ""}
```